chain-core/src/fixed/display.rs

```diff
@@ -0,0 +1,445 @@
+// Copyright © 2018–2020 Trevor Spiteri
+
+// This library is free software: you can redistribute it and/or
+// modify it under the terms of either
+//
+//   * the Apache License, Version 2.0 or
+//   * the MIT License
+//
+// at your option.
+//
+// You should have recieved copies of the Apache License and the MIT
+// License along with the library. If not, see
+// <https://www.apache.org/licenses/LICENSE-2.0> and
```

Comment:
https://github.com/crypto-com/chain/blob/master/client-core/src/hd_wallet/mod.rs#L3

I guess it the redistribution copyright can be licensed under the Apache;
if there were any modifications to the original, also add our modifications copyright to make the license information clear

Reply:
Done